client/src/language/diagnosticsSupport.ts

```diff
@@ -277,6 +278,33 @@ const checkIsIgnoredShellcheckSc2154 = async (
   return commonDirectoriesVariables.has(variableName as string)
 }
 
+const fixDiagnosticMessagePythonImport = (
+  diagnostic: vscode.Diagnostic
+): string => {
+  // Some imports are made automatically by BitBake. Reimporting them produces a warning with irrelevant line number.
+  const fix = (regex: RegExp): string => {
+    const magicLineNumber = 6 // In the embedded language document, the imports are from that line or before.
```

Comment:
So it's not magic then. You can properly name it: `embeddedLanguageDocImportLine`

Reply:
I wanted to emphasize that the number is not procedurally calculated and will break if the number of lines in the header is modified. But yeah, seems like the term was used improperly.